2-mandatory/3-financial-times.js

```diff
@@ -14,7 +14,15 @@ function potentialHeadlines(allArticleTitles) {
     (you can assume words will always be seperated by a space)
 */
 function titleWithFewestWords(allArticleTitles) {
-    // TODO
+    let myMin = 100;
```

Comment:
what would happen if all the titles have more than 100 characters?

Reply:
it will not work, I can put assign 1000 to myMin.
Is there should be another approach

Reply:
you could have something like:
```
function titleWithFewestWords(allArticleTitles) {
    let myMin ;
    let myStr = ''
    for (i in allArticleTitles){
    myMin = myMin || allArticleTitles[i].length
...
    ```
this line `myMin = myMin || allArticleTitles[i].length` is almost similar to a ternary statement where you check if myMin exists `then` use myMin `else` use  allArticleTitles[i].length.

so on the first time in the loop myMin is `undefined` and it will take the first loop value, on the following loop it will not override the variable because it already assigned.

Reply:
Thank you, fixed it now

Reply:
🚀